### DISCUSSION TITLE
Scrolling after the first render

### DISCUSSION THREAD

**Discussion by zapupenec** [KingSora/OverlayScrollbars#555] [Q&A]
Hello!
Trying to use your scrollbar in my study project. Faced a scrolling issue after the first render.
I was looking at the example https://stackblitz.com/edit/overlayscrollbars-react?file=src%2FApp.tsx
The ```scrollContent``` function has a condition ```if (!osInstance) return;``` to which I cannot scroll.
I want to scroll the content to the bottom the first time the application loads. But since ```osInstance=null```, I can't access ```scrollOffsetElement```. I'm just learning and maybe I couldn't figure out how to solve this in react or I missed some overlayscroll parameter. Is there a way to accomplish this?
Sorry for my English) I use a translator.

**Comment by KingSora** [comment]
@zapupenec Good day :)

There are multiple ways to accomplish this, but the easiest one should be to use the `initialized` event which gets fired once the plugin:

```js
<OverlayScrollbarsComponent
  events={{
    initialized: (instance) => {
      instance.elements().viewport.scrollTo(9999, 9999);
    },
  }}
  defer
>
  Content
</OverlayScrollbarsComponent>
```

**Reply by zapupenec** [reply]
This helped with the initialization but ran into another issue. I had to use another heavier sidebar. Maybe later I'll make an example in the sandbox to show you. Thank you very much for your answer.
